src/GLideNUI/configDialog.ui

```diff
@@ -208,6 +208,16 @@
             </widget>
            </item>
            <item>
+            <widget class="QCheckBox" name="vSyncCheckBox">
+             <property name="toolTip">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vertical sync, or VSync, can improve the image by syncing the game's frame rate to your monitor's refresh rate. This prevents image tearing, but may cause performance problems.&lt;/p&gt;&lt;p&gt;[Recommended: &lt;span style=&quot; font-style:italic;&quot;&gt;Usually off, on if you have no problems&lt;/span&gt;]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
You're right. The wording is wrong. I'll probably think about it more later, but I changed it for now.